osf/models/metaschema.py

```diff
@@ -50,14 +50,19 @@ def get_latest_version(self, name):
         """
         return self.filter(name=name).order_by('schema_version').last()
 
-    def get_latest_versions(self, request=None, invisible=False):
+    def get_latest_versions(self, request=None, invisible=False, default_schema_id=None):
         """
         Return the latest version of the given schema
 
         :param request: the request object needed for waffling
         :return: queryset
         """
-        queryset = self.order_by('name', '-schema_version').distinct('name')
+        annotated_qs = self.annotate(default_schema_ordering=models.Case(
```

Comment:
Thank you for finding this Case functionality. I was assuming that my gut instinct of `annotate(is_default=F(id) == default_schema)` wasn't going to work, and the way we were doing it for the licenses made me very sad.